app/views/api/v2/smart_proxies/main.json.rabl

```diff
@@ -2,11 +2,15 @@ object @smart_proxy
 
 extends "api/v2/smart_proxies/base"
 
-attributes :created_at, :updated_at
+attributes :created_at, :updated_at, :hosts_count
 
 child :smart_proxy_features => :features do
   attributes :capabilities
   glue :feature do
     attributes :name, :id
   end
 end
+
+if @status
+  node(:status) { |smart_proxy| smart_proxy.ping ? "OK" : "FAIL" }
```

Comment:
I guess it's better to re-use constants here:
https://github.com/theforeman/foreman/blob/989e0aad01df80dea8fd4734a78e794770b39f2d/app/services/ping.rb#L2-L3